AGENTS.md

```diff
@@ -287,6 +287,26 @@ make lint
 
 ## Best Practices for AI Agents
 
+### Ethical Guidelines
+
+**IMPORTANT: Never Reference Real People**
+
+Plugins, commands, skills, and hooks must NEVER reference real people by name, even as stylistic examples (e.g., "in the style of <specific human>").
+
+**Ethical rationale:**
+1. **Consent**: Individuals have not consented to have their identity or persona used in AI-generated content
+2. **Misrepresentation**: AI cannot accurately replicate a person's unique voice, style, or intent
+3. **Intellectual Property**: A person's distinctive style may be protected
+4. **Dignity**: Using someone's identity without permission diminishes their autonomy
+
+**Instead, describe specific qualities explicitly:**
+
+* "Write commit messages that are direct, technically precise, and focused on the rationale behind changes"
+* "Explain using clear analogies, a sense of wonder, and accessible language for non-experts"
```

Comment:
What do we mean by "a sense of wonder" here? Also, I don't think these bullet points need to be encapsulated by quotation marks.

Reply:
Ohhh, I see now that you're intending this subheading to be examples of specific descriptions. I thought this was like a list of guidelines you wanted people to follow. In which case, the quotation marks do make sense. I think adding "for example:" would help clarify the intention here.

Reply:
Updated